package.json

```diff
@@ -4,13 +4,13 @@
   "description": "Use this to save yourself some time when working on a webpack configuration.",
   "main": "index.js",
   "scripts": {
-    "cover": "nyc --reporter=lcov --reporter=text npm t",
+    "cover": "nyc --reporter=lcov --reporter=text ava --coverage",
```

Comment:
Similar fix for https://github.com/kentcdodds/configuration-validator/issues/9 
Please refer https://github.com/kentcdodds/configuration-validator/issues/9#issuecomment-198073323